controlpanel/frontend/jinja2/base.html

```diff
@@ -83,7 +84,6 @@
         "text": "Home",
         "href": home_url,
         "active": page_name == "home",
-        "admin": request.user.is_superuser,
```

Comment:
I think as part of removing this we might as well also remove the associated bit of code in the template that is rendered. Looks to be in `controlpanel/frontend/static/components/navbar/macro.html`:

```
{% if item.admin %}&#128295;{% endif %}
```

Unless you wanted to leave this in for some reason?